github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -2347,6 +2347,58 @@ periodics:
         privileged: true
     nodeSelector:
       type: bare-metal-external
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: kubevirt-prow-workloads
+  cron: 30 0,7 * * *
```

Comment:
Why are we only testing twice a day here? Also I find the distance of seven hours a bit odd. What's the reason for this?

Reply:
I just thought that twice daily would be enough for this lane
 
I  am open to suggestions otherwise though.